.github/workflows/run_tests.yml

```diff
@@ -15,26 +15,26 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: ["ubuntu-latest", "macos-latest"]
-        # Remove Python 3.7.13 on 27 Jun 2023: https://endoflife.date/python
-        python-version: ["3.7.13", "3.8.13", "3.9.13", "3.10.5"]
+        os: ["ubuntu-20.04", "macos-latest", "windows-latest"]
+        # Remove Python 3.7.9 on 27 Jun 2023: https://endoflife.date/python
+        python-version: ["3.7.9", "3.8.10", "3.9.13", "3.10.8"]
         pandas-version: ["1.0.5", "1.1.5", "1.2.5", "1.3.5", "1.4.3", ""]
 
         exclude:
           # Pandas 1.4.3 requires Python >= 3.8
-          - python-version: "3.7.13"
+          - python-version: "3.7.9"
             pandas-version: "1.4.3"
           # Pandas 1.0.5 has to be fully rebuilt with Python >= 3.9.13 (taking > 10 min)
           - python-version: "3.9.13"
             pandas-version: "1.0.5"
           # Pandas 1.0.5 has to be fully rebuilt with Python >= 3.9.13 (taking > 10 min)
-          - python-version: "3.10.5"
+          - python-version: "3.10.8"
             pandas-version: "1.0.5"
           # Pandas 1.1.5 has to be fully rebuilt with Python >= 3.10.5 (taking > 10 min)
-          - python-version: "3.10.5"
+          - python-version: "3.10.8"
             pandas-version: "1.1.5"
           # Pandas 1.2.5 has to be fully rebuilt with Python >= 3.10.5 (taking > 10 min)
```

Comment:
are these comments no longer accurate or do we just not care about building time?

Reply:
Hum since there is inequality and not strict equality, I guess these comments are still valid.
Example:

```
# Pandas 1.1.5 has to be fully rebuilt with Python >= 3.10.5 (taking > 10 min)
```

Something which is valid for `python >= 3.10.5` is also valid for `python >= 3.10.8`, mathematically speaking :)